tests/std/tests/P0220R1_string_view/test.cpp

```diff
@@ -69,8 +69,7 @@ const evil_conversion_to_string_view_lvalue_only convert_lvalue_only{};
 #pragma clang diagnostic pop
 #endif // __clang__
 
-// We provide a non-standard guarantee that basic_string_view is trivially copyable;
-// P2251 may eventually standardize that guarantee.
+// Since P2251R1 basic_string_view is guaranteed to be trivially copyable.

```

Comment:
Ditto "we don't need to document where this requirement came from as long is it's in the working draft."
```suggestion
```

Reply:
Agreed, but the comment below needs to be updated: `// Similar non-standard guarantees`

Reply:
Upon further thought, because the "similar" guarantees are implied by different Standardese, I think it's reasonable for the test code to cite both paragraphs. Product code usually doesn't extensively cite the Standard for why it's doing ordinary things (only when things are extremely subtle), but test code is typically much more verbose and there's little harm in over-citing the Standard there.